primitives/src/locktime/relative.rs

```diff
@@ -64,6 +81,12 @@ impl LockTime {
     /// let n_lock_time: u32 = 7000;
     /// let lock_time = LockTime::from_consensus(n_lock_time).unwrap();
     /// assert_eq!(lock_time.to_consensus_u32(), n_lock_time);
+    ///
+    /// // However, not all values will round-trip with `to_consensus_u32`.
+    /// let n_lock_time: u32 = 0xfffff;
+    /// let lock_time = LockTime::from_consensus(n_lock_time).unwrap();
+    /// assert_ne!(lock_time.to_consensus_u32(), n_lock_time);
+    /// assert_eq!(lock_time.to_consensus_u32(), 0xffff);
```

Comment:
I added #3970 if you are interested @jamillambert. Probably wait for @apoelstra to comment before starting in case I've got the wrong end of the stick.

General points

- I all examples I rekon if its not too much work / noise we should use `?` not `unwrap`
- IMO we should lead users into using `relative::LockTime` and `relative::Height` not importing and plain old `LockTime`
- Perhaps put a code comment on any hardcoded value because I don't know what the significance of `0xffff` is or if there is any significance. 
- Have to remember that this locktime stuff is painfully easy to get mixed up so examples should be readable by beginners and by advance guys who can't remember - which means everyone.

Reply:
> * IMO we should lead users into using `relative::LockTime` and `relative::Height` not importing and plain old `LockTime`

Yes +1 for this. This is how I always use the `*::LockTime` stuff.

Reply:
> * IMO we should lead users into using `relative::LockTime` and `relative::Height` not importing and plain old `LockTime`

I'll do this in a follow up PR.  None of the existing Examples did it this way so needs to be changed everywhere.
Opened Issue #3976 so it's not forgotten.

Reply:
I have changed `unwrap` to `?`

I also changed the hard coded example times and blocks to more meaningful values and added comments explaining what they are.